dialog/src/main/java/roboy/ros/RosMainNode.java

```diff
@@ -413,6 +413,7 @@ public boolean ApplyFilter(String filterName) {
 
         if(services.notInitialized(RosServiceClients.SNAPCHATFILTER)) {
             // FALLBACK RETURN VALUE
+            LOGGER.info("ApplyFilter not initialized");
```

Comment:
> Applies to all instances of Logger.info

How important is this info. Considering Logging is going to be split and there will be options to set logging levels, would you consider this a good to know `info` that one could ignore in regular operations or something the user should be warned about to fix when it comes up (`LOGGER.warn`)?

https://logging.apache.org/log4j/log4j-2.1/manual/customloglevels.html